hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -151,6 +152,7 @@
       reset_connections: {rst_ni: "lc"},
       type: "eflash",
       base_addr: "0x20000000",
+      swaccess: "ro",
```

Comment:
@eunchan please review the access type I added in this file, and the update in topgen and reggen.

Reply:
Seems correct.

Reply:
Thanks for confirming.